src/main/java/Utils/Tables.java

```diff
@@ -39,6 +39,20 @@ public static void createUserTable() {
             );
             UserDao.insertUser(admin);
             System.out.println("Admin user created successfully!");
+            
+            // staff test user
+            Tuple<byte[], byte[]> saltAndHash1 = createSaltAndHash("staff");
```

Comment:
This will need to be removed - we only want to create a base admin class, none others. Staff will be added after deployment.

Reply:
I've only kept this in for testing purposes. Will be deleted however when possible

Reply:
Commented out!